From: Pietro Battiston <me@pietrobattiston.it>
Date: Mon, 9 Apr 2012 11:49:47 +0200
Subject: Fix for Debian bug #666655.

The test for cascaded_union involves scattering random patches and
verifying they form a MultiPolygon: depending on the random
scattering, they can instead (rarely) form a Polygon, and in that
case the test fails. This is a (hacky?) patch solving the problem.
---
 shapely/tests/cascaded_union.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/shapely/tests/cascaded_union.txt b/shapely/tests/cascaded_union.txt
index 7d54f31..2446349 100644
--- a/shapely/tests/cascaded_union.txt
+++ b/shapely/tests/cascaded_union.txt
@@ -19,6 +19,7 @@ Buffer the points, producing 100 polygon spots
 Perform a cascaded union of the polygon spots, dissolving them into a 
 collection of polygon patches
 
-  >>> cascaded_union(spots) # doctest: +ELLIPSIS
-  <shapely.geometry.multipolygon.MultiPolygon object at 0x...>
+  >>> u = cascaded_union(spots) # doctest: +ELLIPSIS
+  >>> u.__class__.__name__.endswith( 'Polygon' )
+  True
 
